app/src/main/java/com/android/sample/model/recipe/networkData/FirestoreRecipesRepository.kt

```diff
@@ -305,17 +305,20 @@ class FirestoreRecipesRepository(private val db: FirebaseFirestore) : RecipeNetw
     var finalQuery: Query = db.collection(FIRESTORE_COLLECTION_NAME)
 
     /** ******************** Apply all the filters ********************* */
-
     /*
-     * Category filter
+    If you decide to change or add a filter in the future, you can add it here.
```

Comment:
- **Documentation**: The added links to the wiki are great for onboarding! Just make sure the wiki stays updated. 👍

- **Category Filter**: Nice use of `.let`—clean and easy to follow. No issues here.

- **Time Filter**: `takeIf` is a solid choice for validation. Just double-check edge cases where `timeRange.min` or `max` hit the boundaries.

- **Difficulty Filter**: The `Difficulty.Undefined` check is clear and works well. Consistency across filters looks good too.

Reply:
Thanks